### ISSUE TITLE
Change a payment method screen

### ISSUE THREAD

**Issue by mp3il** [bluesnap/bluesnap-ios-int#18]
Hi,

Does the SDK allow us to show a "Choose your payment method" screen? without a "Pay $.." button? just "save" and we get a pftoken we can store and save as preferred method?

**Comment by snpori** [comment]
We have a returning shopper flow that you can use to just tokenize the card and use it later. Is this what you mean?

**Comment by mp3il** [comment]
Not exactly, I do use the returning shopper flow - but I want to use the SDK just to show a "Select a payment method".

In the current UI you have to pass a Checkout screen with an amount and it shows a "Pay ..." Button

I'm asking if there is a way to just show the "Select your payment method" without the pay button.

**Comment by snpori** [comment]
It sounds like you are trying to create a subscription flow or something similar to subscriptions

The subscription flows (BSSdkRequestSubscriptionCharge as mentioned in the readme) should not show a pay button since they are supposed to only create a subscription.
